coalib/processes/Processing.py

```diff
@@ -385,8 +396,16 @@ def instantiate_processes(section,
     # the whole project is accessible to every bear. However, local bears are
     # run only for the changed files if caching is enabled.
 
+    if len(use_raw_files) > 1:
+        log_printer.err("Bears that uses raw files can't be mixed with "
+                        'Bears that uses text files. Skipping this '
+                        'section.')
```

Comment:
I think it's crucial here to propose a solution to the user. E.g. "Move the following bears into an own section: ..."

The user doesn't care if a bear uses raw files or not and he also doesn't know so this message needs to be very specific about how to solve this IMO